dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/companies/CompanyInformation.kt

```diff
@@ -25,6 +26,8 @@ data class CompanyInformation(
 
     val companyAlternativeNames: List<String>?,
 
+    val companyContactDetails: List<String>?,
```

Comment:
Wird eigentlich irgendwo validiert, dass hier auch nur valide Emails eingegeben werden?
Oder ist das wirklich ein free-text-string?

Reply:
we have no validation rule for the content of this field